web/global.d.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * @copyright 2016-present Kriasoft (https://git.io/vMINh)
+ */
+
+import {} from "react";
+import { Theme } from "@emotion/react";
+import { Interpolation } from "@emotion/serialize";
```

Comment:
isn't this type reexported from `@emotion/react`? I don't think you should depend on `@emotion/serialize`, it won't cause problems but it's really just an internal package for the most part, so people should not be expected to interact with it directly

Reply:
Good point.